networkx/generators/geometric.py

```diff
@@ -72,8 +72,6 @@ def random_geometric_graph(n, radius, dim=2, pos=None, metric=None):
         If this argument is not specified, the Euclidean distance metric is
         used.
 
-    .. _metric: https://en.wikipedia.org/wiki/Metric_%28mathematics%29
```

Comment:
I assumed this would parse correctly when I created this. Maybe I need to indent this so that it is at the parameter definition? I assumed it would be parsed as part of the "Parameters" section. Anyway, thanks for fixing it!